docs/reference/glossary.asciidoc

```diff
@@ -188,6 +188,12 @@ Frozen indices use a memory-efficient shard implementation and throttle searches
 Searching a frozen index is lower overhead than re-opening a closed index to enable searching.
 // end::frozen-index-def[]
 
+[[glossary-hidden-index]] hidden index ::
+// tag::hidden-index-def[]
+An index that is excluded by default when you access indices using a wildcard expression. 
+To include hidden indices, you must specify the `expand_wildcards` parameter.
```

Comment:
One minor point that it might be good to mention here: Hidden indices are also included for wildcard expressions that start with a dot, e.g. `.watcher-history*`. This was done in part to mirror the Unix globbing behavior for dot files.